pages/blog/announcing-graphql-yoga-2.mdx

```diff
@@ -0,0 +1,278 @@
+
+Here, we are building a full-featured GraphQL API with validation cache, security rules, and tracing with only a few lines of code:
+
+```tsx
+import { createServer } from '@graphql-yoga/node'
+
+import { useDepthLimit } from '@envelop/depth-limit'
+import { useResponseCache } from '@envelop/response-cache'
+import { useSentry } from '@envelop/sentry'
+
+import { schema } from './schema'
+
+const server = createServer({
+  schema: {
+    typeDefs: /* GraphQL */ `
+      type Query {
+        hello: String
+      }
+    `,
+    resolvers: {
+      Query: {
+        hello: () => 'Hello Hello Hello'
+      }
+    }
+  },
+  plugins: [
+    useDepthLimit({
+      // set up some security rules
+      maxDepth: 10
+    }),
+    useResponseCache(), // speed up our server with a response cache
+    useSentry() // report unexpected errors to sentry
+  ]
+})
+
+// Start the server and explore http://localhost:4000/graphql
+server.start()
+```
+
+The Envelop Plugin currently [proposes more than 35+ plugins](https://www.envelop.dev/plugins) covering most of the standard GraphQL APIs features.
+
+Ultimately, you can [develop custom Envelop plugins](https://www.envelop.dev/docs/plugins/custom-plugin) to create reusable behaviors that hook on to the GraphQL lifecycle.
+
+<p>&nbsp;</p>
+
+### Production-ready
+
+GraphQL Yoga v2 has been built in production for production usage.
+
+Built-in real-world conditions within our projects (ex: GraphQL Mesh) and with some of [our clients](https://www.the-guild.dev/services), performance was highly prioritized.
+The core of Yoga is as performant as possible, and we are constantly trying to improve it.
+
+Also, the Yoga V2 repository runs performance checks on every commit and Pull Request, so we can always capture any performance regression.
+
+We continue our effort of pushing GraphQL Yoga to more production environments with the imminent release of Redwood 1.0 that will use Yoga 2.0 as the default GraphQL server.
+
+<p>&nbsp;</p>
+
+### A standards-compliant GraphQL Server
+
+In the same way, as TypeScript aims to stay aligned with ECMAScript, GraphQL Yoga is based on several official and recognized specs:
+
+- [GraphQL-spec](https://spec.graphql.org/October2021/), [GraphQL-over-HTTP](https://github.com/graphql/graphql-over-http): guarantees your GraphQL API to work with all existing GraphQL clients (Apollo, Relay, URQL, and more).
+- [GraphQL-Multipart-Request](https://github.com/jaydenseric/graphql-multipart-request-spec): enables great File uploads support.
+- [W3C Fetch API](https://fetch.spec.whatwg.org/): we embrace the future of Node.js and provide the same developer experience on all platforms.
+
+Yoga v2 also supports some experimental GraphQL features such as [`@defer` and `@stream`](https://github.com/graphql/graphql-js/pull/2839), allowing you to get a taste of the future of GraphQL (with compatible clients such as [URQL](https://formidable.com/open-source/urql/docs/comparison/#core-features)).
```

Comment:
I would add that as a section, here is a rough draft you can merge with what you wrote already:

### GraphQL Features from the future

Thanks to the Envelop plugin system, Yoga v2 can also act as `"Babel for GraphQL"`, giving you the option to use features that are not yet in the GraphQL spec but are very useful in production today, like defer and stream and [OneOf](https://www.envelop.dev/docs/guides/using-graphql-features-from-the-future)